connectors/es/management_client.py

```diff
@@ -63,30 +63,29 @@ async def create_content_index(self, search_index_name, language_code):
             )
         )
 
-    async def ensure_content_index_mappings(self, index, mappings):
+    async def ensure_content_index_mappings(self, index_name, index, desired_mappings):
```

Comment:
we have both `index` and `index_name` in args? 

can we add docstrings that `index_name` is `str` (and it can be an alias?) 
and `index` is `object` representaiton of index

Reply:
`index_name` is just a name and `index` its definition (mappings/settings).

It's not the best, but it follows the same logic another method in the class has:

```
    async def ensure_content_index_settings(
        self, index_name, index, language_code=None
    ):
```

We can refactor/rename, but in a different PR. I'd lazily keep it as is - methods are small and stable